t/133-balancer.t

```diff
@@ -7,9 +7,9 @@ use Test::Nginx::Socket::Lua;
 #workers(2);
 #log_level('warn');
 
-repeat_each(2);
+repeat_each(1);
```

Comment:
The new test cases can be passed only the first time, like the warn string: "load balancing method redefined"